dts/bindings/flash_controller/st,stm32-ospi-nor.yaml

```diff
@@ -32,6 +32,9 @@ properties:
   reset-gpios:
     type: phandle-array
     description: RESETn pin
+ reset-gpios-duration:
```

Comment:
also to be included in the dts/bindings/flash/flash-controller/st,stm32-xspi-nor.yaml

Reply:
@FRASTM The xpsi is including this file, and then making alterations, so only the ospi is needed from what I understand.
Ref: https://github.com/zephyrproject-rtos/zephyr/blob/153f5dad050b0e57cdc0b9f4cde51df0d7c0bd0c/dts/bindings/flash_controller/st%2Cstm32-xspi-nor.yaml#L21